app/models/user_target.rb

```diff
@@ -1,10 +1,16 @@
 class UserTarget < ApplicationRecord
+  acts_as_mappable default_units: :meters,
+                   lat_column_name: :latitude,
+                   lng_column_name: :longitude
   belongs_to :user
   belongs_to :topic
 
   validates :title, :user, :topic, :latitude, :longitude, :area, presence: true
   validate :target_limit
 
+  scope :not_from_user, lambda { |user_id| where('user_id != ?', user_id) }
```

Comment:
Check the syntax for the scopes: http://edgeguides.rubyonrails.org/active_record_querying.html#passing-in-arguments